docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,280 @@
+
+The following is an example modal bottom sheet:
+![A modal bottom sheet with share, link, edit, and delete list items](assets/bottom_sheets/SheetsBottom_modal_screenshot.jpg)
+
+```dart
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Standard Bottom Sheet Demo',
+      home: Scaffold(
+        // Use a Builder to get a context within the Scaffold.
+        body: Builder(
+          builder: (context) {
+            return Center(
+              child: FlatButton(
+                child: Text('Show Bottom Sheet'),
+                onPressed: () {
+                  showModalBottomSheet(
+                    context: context,
+                    builder: (context) {
+                      // Using Wrap makes the bottom sheet height the height of the content.
+                      // Otherwise, the height will be half the height of the screen.
+                      return Wrap(
+                        children: [
+                          ListTile(
+                            leading: Icon(Icons.share),
+                            title: Text('Share'),
+                          ),
+                          ListTile(
+                            leading: Icon(Icons.link),
+                            title: Text('Get link'),
+                          ),
+                          ListTile(
+                            leading: Icon(Icons.edit),
+                            title: Text('Edit name'),
+                          ),
+                          ListTile(
+                            leading: Icon(Icons.delete),
+                            title: Text('Delete collection'),
+                          ),
+                        ],
+                      );
+                    },
+                  );
+                },
+              ),
+            );
+          },
+        ),
+      ),
+    );
+  }
+}
+```
+
+## Anatomy and key properties
+
+The following shows the anatomy of a modal bottom sheet:
+
+![Modal bottom sheet anatomy](assets/bottom_sheets/modal-anatomy.png)
+
+1. Sheet
+2. Contents
+3. Scrim
+
+
+#### Sheets properties
+
+&nbsp;         | Property
+-------------- | ------------------------
+| **Color** | `backgroundColor` |
+| **Elevation** | `elevation` |
+| **Shape** | `shape` |
+
+#### Contents properties
+
+There are no specific properties for content because the content can be any composition of widgets.
+
+#### Scrim properties
+
+&nbsp;         | Property
+-------------- | ------------------------
+| **Color** | `barrierColor` |
+
+
+## Expanding bottom sheet
+
+Expanding bottom sheets require creating a custom widget. See [Expanding bottom sheet](link) for more info.
+TODO: link https://github.com/material-components/material-components-flutter/blob/develop/docs/components/expanding-bottom-sheet.md on material.io
```

Comment:
I don't know about having the TODO

Reply:
Go ahead and link to the the github page.